Content.Shared/Animals/WoolyComponent.cs

```diff
@@ -1,17 +1,16 @@
-using Content.Server.Animals.Systems;
 using Content.Shared.Chemistry.Components;
 using Content.Shared.Chemistry.Reagent;
 using Content.Shared.FixedPoint;
+using Robust.Shared.GameStates;
 using Robust.Shared.Prototypes;
-using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
 
-namespace Content.Server.Animals.Components;
+namespace Content.Shared.Animals;
 
 /// <summary>
-///     Lets an entity produce wool fibers. Uses hunger if present.
+///     Gives the ability to produce wool fibers;
+///     produces endlessly if the owner does not have a HungerComponent.
 /// </summary>
-
-[RegisterComponent, Access(typeof(WoolySystem))]
+[RegisterComponent, Access(typeof(WoolySystem)), AutoGenerateComponentPause, NetworkedComponent]
```

Comment:
```suggestion
[RegisterComponent, AutoGenerateComponentState, AutoGenerateComponentPause, NetworkedComponent]
```
let's make this identitcal to UdderComponent
having access to some of these should be totally fine

Please adjust the DataFields below to have the same networked and access attributes as their equivalents in UdderComponent.